tox.ini

```diff
@@ -0,0 +1,15 @@
+[flake8]
+# C0111: docstrings
+# I0011: locally disabled warnings
+# R0801: similar lines
+# R0904: too many public methods
+# R0911: too many return statements
+# R0912: too many branches
+# R0913: too many arguments
+# R0903: too few public methods
+# W0141: used builtin 'map' function
+# W0142: used star magic
+# W0212: access to a protected member
+ignore = C0111,I0011,R0801,R0904,R0911,R0912,R0913,R0903,W0141,W0142,W0212
+exclude = .git,__pycache__
+max-line-length = 82
```

Comment:
I've followed [DNF hacking guide](https://github.com/rpm-software-management/dnf/wiki/Hacking#coding-style) and also copied it from [pylint script](https://github.com/rpm-software-management/dnf/blob/master/scripts/dnf_pylint). If it would be just on my preference I would kept the default value. This topic is subjective but generally in typography the shorter the line is, the more readable paragraph is.